vertx-auth-oauth2/src/main/java/io/vertx/ext/auth/oauth2/DCROptions.java

```diff
@@ -0,0 +1,52 @@
+package io.vertx.ext.auth.oauth2;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.codegen.json.annotations.JsonGen;
+import io.vertx.core.http.HttpClientOptions;
+import io.vertx.core.json.JsonObject;
+
+@DataObject
+@JsonGen(publicConverter = false)
+public final class DCROptions {
+
+  /**
+   * The base url of the OIDC provider like Keycloak.
+   */
+  private String site;
+
+  /**
+   * Name of the tenant if any. Keycloak call this realm.
+   */
+  private String tenant;
```

Comment:
usually we have getters and setters for these, to make it more idiomatic from java.